pal/samples/esp8266/telemetry_sample.c.ino

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <AzureIoTUtility.h>
+
+// CAVEAT: This sample is to demonstrate azure IoT client concepts only and is not a guide design principles or style
+// Checking of return codes and error values shall be omitted for brevity.  Please practice sound engineering practices
+// when writing production code.
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "iothub.h"
+#include "iothub_client.h"
+#include "iothub_device_client_ll.h"
+#include "iothub_client_options.h"
+#include "iothub_message.h"
+#include "azure_c_shared_utility/crt_abstractions.h"
+#include "azure_c_shared_utility/shared_util_options.h"
+#include "azure_c_shared_utility/tlsio.h"
+#include "azure_c_shared_utility/xlogging.h"
+
+#define SET_TRUSTED_CERT_IN_SAMPLES
+#ifdef SET_TRUSTED_CERT_IN_SAMPLES
+#include "certs/certs.h"
+#endif // SET_TRUSTED_CERT_IN_SAMPLES
+
+// You must set the device id, device key, IoT Hub name and IotHub suffix in
+// iot_configs.h
+#include "iot_configs.h"
+#include "sample_init.h"
+
+static char ssid[] = IOT_CONFIG_WIFI_SSID;
+static char pass[] = IOT_CONFIG_WIFI_PASSWORD;
+
+#ifdef SAMPLE_MQTT
+    #include "AzureIoTProtocol_MQTT.h"
+    #include "iothubtransportmqtt.h"
+#endif // SAMPLE_MQTT
+#ifdef SAMPLE_HTTP
+    #include "AzureIoTProtocol_HTTP.h"
+    #include "iothubtransporthttp.h"
+#endif // SAMPLE_HTTP
+
+/* This sample uses the _LL APIs of iothub_client for example purposes.
+Simply changing the using the convenience layer (functions not having _LL)
+and removing calls to _DoWork will yield the same results. */
+
+/* Paste in the your iothub connection string  */
```

Comment:
>the [](start = 12, length = 3)

typo
looks like this is in the c sdk as well

Reply:
actually this should probably be removed since you don't actually paste in your connection string

---
In reply to: [342692575](https://github.com/Azure/azure-iot-pal-arduino/pull/7#discussion_r342692575) [](ancestors = 342692575)

Reply:
Done.